src/draw.rs

```diff
@@ -99,7 +85,7 @@ impl Font {
     pub fn add_str_to_cache(&mut self, s: &str) {
         for ch in s.chars() {
             if self.glyphs.get(&ch).is_none() {
-                let glyph = CachedGlyph::new(self.font, self.size, ch);
+                let glyph = CachedGlyph::new(self.font.clone(), self.size, ch);
```

Comment:
Unwanted clone - this one leads to a new font copy every time we prime a font cache.